caffe2/mobile/contrib/arm-compute/README.md

```diff
@@ -0,0 +1,29 @@
+# Caffe2 - ARM Compute Backend
+
+## Build
+
+To build, clone and install scons
+
+```
+brew install scons
+```
+
+use the build\_android.sh:
```

Comment:
typo?

Reply:
Which word? build\_android.sh? that is probably for markdown..